app/models/post.rb

```diff
@@ -2,4 +2,18 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   has_many :comments
   has_many :likes
+
+  def update_post_counter
+    post_counter = User.find(author_id).posts_counter
+    if post_counter.nil?
+      post_counter = 1
+    else
+      post_counter += 1
+    end
+    User.find(author_id).update(posts_counter: post_counter)
+  end
```

Comment:
[OPTIONAL]
```suggestion
  after_save :update_post_counter
  
  private
  
  def update_post_counter
    post_counter = User.find(author_id).posts_counter
    if post_counter.nil?
      post_counter = 1
    else
      post_counter += 1
    end
    User.find(author_id).update(posts_counter: post_counter)
  end
```
- [ ] Nice work implementing the `update_post_counter` method :clap: However it would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private